### ISSUE TITLE
[Solved][Feature Request] Options to stay focused on Tree buffer after selection

### ISSUE THREAD

**Issue by RaZ0rr-Two** [nvim-tree/nvim-tree.lua#716]
An option to remain on the tree buffer after selecting a file would be nice. This will allow selecting multiple files easily. Would compliment the `let g:nvim_tree_quit_on_open = 0` option. 

Also, an alternative would be `cb = tree_cb("select") }` or `cb = tree_cb("tag") }`with which multiple files could be tagged for operation (opening, deleting) later.

**Comment by kyazdani42** [comment]
what do you mean by selecting ? Could this be related to #612 ?

**Comment by RaZ0rr-Two** [comment]
Also, closing comment.

If possible to change, then maybe `tree_cb("preview")` can be changed to something like `tree_cb("openbuffer")` to avoid confusion

**Comment by anthony-S93** [comment]
> 

I noticed that the <Tab> key can open only one file at most. That means if I press <Tab> on, say, `foo1.txt` and then navigate to `foo2.txt` and press <Tab> again, `foo2.txt` will open as expected but `foo1.txt` will not persist.  

Here is a couple of screenshots to illustrate this behavior: 
1) Two files `foo1.txt` and `foo2.txt` located in the same directory. No files are opened yet. 
![2023-02-18-035736_721x374_scrot](https://user-images.githubusercontent.com/69449791/219779768-bcf5ac6f-dd22-4ef9-8665-f8d6cc2bcd87.png)

2) Open up `nvim-tree`. Navigate cursor to `foo1.txt` and press <Tab>. `foo1.txt` opens in a buffer as expected. 
![2023-02-18-035947_734x370_scrot](https://user-images.githubusercontent.com/69449791/219780287-cf079a37-89fe-4072-80d3-e14153c7cc20.png)

3) Next, navigate the cursor to `foo2.txt`, and then hit <Tab> again. 
![2023-02-18-040035_661x316_scrot](https://user-images.githubusercontent.com/69449791/219780601-129ed152-40f0-4be4-a8b5-504170a1c2f2.png)

**Result: `foo2.txt` opens, but `foo1.txt` disappears.** 

Is that by design?

My workflow with `nvim-tree` usually comprises of:
1) Pressing a keybinding that runs `:NvimTreeToggle`
2) Select file with return key (`nvim-tree` closes because I set `quit_on_open` to `true`).
3) If I need to open another file, I go back to step 1. 

I'd prefer it if there is a way to (as suggested by @RaZ0rr-Two previously) keep the focus on `nvim-tree` after we open a file and to keep the focus on the tree until we close the tree with `:NvimTreeToggle`. That way, we are free to use `nvim-tree` to open any number of files we like without having to switch focus back and forth between `nvim-tree` and the editor. 

At first, I thought that pressing the <Tab> key was the answer I was looking for, but unfortunately, none of the buffers opened with the <Tab> key stays in memory if we open subsequent files with the <Tab> key.
